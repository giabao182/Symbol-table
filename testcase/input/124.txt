INSERT x number
BEGIN
INSERT x string
BEGIN
INSERT x number
PRINT
END
PRINT
END
PRINT