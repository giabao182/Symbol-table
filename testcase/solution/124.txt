success
Redeclared: INSERT x string
success
x//2
x//0
x//0