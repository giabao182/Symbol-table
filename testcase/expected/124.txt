success
success
success
x//2
x//1
x//0